heliopy/util/config.py

```diff
@@ -22,15 +23,16 @@ def get_config_file():
     config_filename = 'heliopyrc'
 
     # Get user configuration location
-    home_dir = os.path.expanduser("~")
-    config_file_1 = os.path.join(home_dir, '.heliopy', config_filename)
+    home_dir = Path.home()
+    config_file_1 = home_dir / '.heliopy' / config_filename
 
-    module_dir = os.path.dirname(heliopy.__file__)
-    config_file_2 = os.path.join(module_dir, config_filename)
+    module_dir = Path(heliopy.__file__)
+    config_file_2 = module_dir / '..' / config_filename
```

Comment:
Why is the '..' bit here? I think the config file lives inside the module_dir, so this isn't needed.

Reply:
heliopy.__file__ locates to /usr/local/lib/python3.5/dist-packages/heliopy/__init__.py so we need to take back a folder to config file.